eng/Versions.props

```diff
@@ -86,7 +86,7 @@
     <MicrosoftTargetingPackPrivateWinRTPackageVersion>1.0.5</MicrosoftTargetingPackPrivateWinRTPackageVersion>
     <MicrosoftDiaSymReaderNativePackageVersion>1.7.0</MicrosoftDiaSymReaderNativePackageVersion>
     <!-- Infrastructure and test-only. -->
-    <MicrosoftSourceLinkVersion>1.0.0-beta2-19367-01</MicrosoftSourceLinkVersion>
+    <MicrosoftSourceLinkVersion>1.0.0-beta2-18618-05</MicrosoftSourceLinkVersion>
```

Comment:
Do we want to keep the latest SourceLink? It sounds like that should be OK, right?

Reply:
I plan to do new PR for arm64 (hopefully that will not break everything). That PR will re-bump the SourceLink version. Do you think I should keep the bumped sourcelink version here?

Reply:
In theory, I think including this in the revert is nice, so `git blame` on this line will point to the new PR rather than the old reverted one. Simple reverts are easier to keep track of. Very unlikely that will practically matter though.

If there are some practical benefits we've started depending on with the new version, that would be another story, I suppose.